chemclipse/plugins/org.eclipse.chemclipse.chromatogram.msd.classifier/src/org/eclipse/chemclipse/chromatogram/msd/classifier/core/AbstractChromatogramClassifier.java

```diff
@@ -27,10 +28,12 @@
 
 	@Deprecated
 	public AbstractChromatogramClassifier() {
-		this(DataType.MSD);
+
+		this(DataType.MSD, DataType.WSD);
```

Comment:
instead of calling this constructor (what now claims that MSD and WSD is supported for **all** derived types, derived types should simply not call` super()`  constructor (as it is deprecated anyways) but instead call `super(DataType.MSD)` or `super(DataType.WSD)` ... (or even multiple types if desired)